feature-group-definitions/array-at.dist.yml

```diff
@@ -0,0 +1,25 @@
+# Generated from: array-at.yml
+# Do not edit this file by hand. Edit the source file instead!
+# Last modified: 2024-04-04T09:37:18.429438422+00:00[UTC]
+
+name: Array at()
+description: Array at() and typed array at() methods
+spec: https://tc39.es/ecma262/multipage/indexed-collections.html#sec-array.prototype.at
+snapshot: ecmascript-2022
+group:
+  - arrays
+  - typed-arrays
+status:
+  baseline: low
+  baseline_low_date: 2022-03-14
+  support:
+    chrome: "92"
+    chrome_android: "92"
+    edge: "92"
+    firefox: "90"
+    firefox_android: "90"
+    safari: "15.4"
+    safari_ios: "15.4"
+compat_features:
+  - javascript.builtins.Array.at
+  - javascript.builtins.TypedArray.at
```

Comment:
This file demonstrates dist-ing a feature that already has both a status and `compat_features` list. We don't do anything here.